src/core/server/server.ts

```diff
@@ -145,6 +145,17 @@ export class Server {
 
     const metricsSetup = await this.metrics.setup({ http: httpSetup });
 
+    const renderingSetup = await this.rendering.setup({
```

Comment:
This change was done to be able to instantiate `rendering` & `httpResourcesSetup` internal services and expose their API to the plugins